docs/schemas/v1/definitions.json

```diff
@@ -0,0 +1,616 @@
+  "properties": {
+    "Page": {
+      "type": "object",
+      "properties": {
+        "apiVersion": {
+          "type": "string",
+          "const": "v1",
+          "description": "Defines the version of this object. Used in determining compatibility between Toolpad \"page\" objects."
+        },
+        "kind": {
+          "type": "string",
+          "const": "page",
+          "description": "Describes the nature of this Toolpad \"page\" object."
+        },
+        "spec": {
+          "type": "object",
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "Serves as a canonical id of the page."
+            },
+            "title": {
+              "type": "string",
+              "description": "Title for this page."
+            },
+            "parameters": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/NameStringValuePair"
+              },
+              "description": "Parameters for the page. These can be set inside of the url query string."
+            },
+            "queries": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "type": "string",
+                    "description": "A name for the query"
+                  },
+                  "enabled": {
+                    "anyOf": [
+                      {
+                        "type": "boolean"
+                      },
+                      {
+                        "$ref": "#/definitions/JsExpressionBinding"
+                      }
+                    ],
+                    "description": "Activates or deactivates the query. When deactivated the data won't be loaded when the page opens."
+                  },
+                  "parameters": {
+                    "type": "array",
+                    "items": {
+                      "type": "object",
+                      "properties": {
+                        "name": {
+                          "type": "string",
+                          "description": "The name"
+                        },
+                        "value": {
+                          "anyOf": [
+                            {},
+                            {
+                              "$ref": "#/definitions/JsExpressionBinding"
+                            }
+                          ],
+                          "description": "The value"
+                        }
+                      },
+                      "required": [
+                        "name"
+                      ],
+                      "additionalProperties": false,
+                      "description": "A name/value pair."
+                    },
+                    "description": "Parameters to pass to this query."
+                  },
+                  "mode": {
+                    "type": "string",
```

Comment:
Not related to this PR: This can probably be better typed.

Reply:
It has 
```json
                   "enum": [
                      "query",
                      "mutation"
                    ],
```
I can improve this in the docs